service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandlerPostDocumentsTest.java

```diff
@@ -45,7 +48,7 @@
     private FurtherEvidenceUploadedEventHandler furtherEvidenceUploadedEventHandler;
 
     @Test
-    void shouldSendDocumentByPostWhenPDFUploadedByRespSolicitor() {
+    void shouldSendDocumentByPostWhenEvidenceBundlePDFUploadedByRespSolicitor() {
```

Comment:
I think it might be better leaving `RespSolicitor` as `RespondentSolicitor` for maintainability. I don't we gain much from shortening the name here and people might be more likely to search the longer name when looking through the code.